Sources/SocialKit/AppRating/AppRatingModifier.swift

```diff
@@ -9,30 +9,52 @@ import SwiftUI
 
 struct AppRatingModifier: ViewModifier {
 
-    private let service: AppRatingService?
-    init(service: AppRatingService? = nil) {
-        self.service = service
+    @ObservedObject private var service: AppRatingService
+    private let style: AppRatingPrePromptStyle?
+    private let closeAction: ((AppRatingPrompt.Outcome)->Void)?
+
+    init(service: AppRatingService?,
+         style: AppRatingPrePromptStyle?,
+         closeAction: ((AppRatingPrompt.Outcome)->Void)? = nil) {
+        self.service = service ?? AppRatingService(forcePresentPrompt: true)
+        self.style = style
+        self.closeAction = closeAction
+
+//        if service?.configuration.appId == nil {
+//            fatalError("Expected a valid AppId")
+//        }
```

Comment:
Useles?

Reply:
I think it should be placed somewhere to safeguard the appId is set before, as it would be quite easy to forget and not obvious to catch the issue. But I didn't figured yet where to place it. I think it would be easier when this feature will be actually implemented. I added a comment to explain this.